### ISSUE TITLE
Install fails on Ubuntu 18.04

### ISSUE THREAD

**Issue by HankB** [foxundermoon/vs-shell-format#26]
```
hbarta@baobabb:~$ mkdir -p /usr/local/bin
hbarta@baobabb:~$ curl -L 'https://github.com/mvdan/sh/releases/download/v2.6.2/shfmt_v2.6.2_linux_amd64' --output  /usr/local/bin/shfmt
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   614    0   614    0     0   1149      0 --:--:-- --:--:-- --:--:--  1149
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0Warning: Failed to create the file /usr/local/bin/shfmt: Permission denied
  0 2826k    0 16360    0     0  12791      0  0:03:46  0:00:01  0:03:45 69617
curl: (23) Failed writing body (0 != 16360)
hbarta@baobabb:~$ chmod a+x /usr/local/bin/shfmt
chmod: cannot access '/usr/local/bin/shfmt': No such file or directory
hbarta@baobabb:~$ echo '**Enjoy shellscript!**'
**Enjoy shellscript!**
hbarta@baobabb:~$ echo 'fork or star  https://github.com/foxundermoon/vs-shell-format'
fork or star  https://github.com/foxundermoon/vs-shell-format
hbarta@baobabb:~$ 
```
/usr/local/bin is not writable by the user unless they are logged in as root. There are a couple possible fixes for this. One is to prefix the command with `sudo` (e.g. `sudo mkdir -p /usr/local/bin`) Which will prompt for the user's password (and will work if they are allowed to use the `sudo` command and it is installed.)
Another - and probably safer - alternative is to put the executable in `~/bin` (expands to `/home/<username>/bin`) and is writable by the user without elevating privileges. If the directory exists, it will probably be in the `$PATH`. If not, it will be there on the next login. If your plugin provides the full path to shfmt, it should always work.

This is probably common to most, if not all, Linux variants.

**Comment by foxundermoon** [comment]
install by yourself and config the path  `"shellformat.path": null`  temporary.

**Comment by francoism90** [comment]
@foxundermoon Please don't do global installs, better is to skip installing this bin. Just provide info on the user how to install pkg.

**Comment by hubitor** [comment]
Same on Arch Linux...

**Comment by foxundermoon** [comment]
resolved on version 6.0.0
